include/eve/module/core/regular/impl/simd/arm/neon/bit_andnot.hpp

```diff
@@ -13,11 +13,12 @@
 
 namespace eve::detail
 {
-template<arithmetic_scalar_value T, typename N>
-EVE_FORCEINLINE wide<T, N>
-                bit_andnot_(EVE_SUPPORTS(neon128_),
-                            wide<T, N> const                &v0,
-                            wide<T, N> const                &v1) noexcept requires arm_abi<abi_t<T, N>>
+template<arithmetic_scalar_value T, typename N, callable_options O>
```

Comment:
this arithmetic_scalar_value is unneeded. No concept in the xxx_ version

Reply:
done